pages/osx/pod.md

```diff
@@ -0,0 +1,31 @@
+# pod
+
+> Dependency manager for Swift and Objective-C Cocoa projects.
+
+- Create a Podfile for the current project with the default contents:
+
+`pod init`
+
+- Download all pod defined in Podfile and create an Xcode Pods library project in `./Pods`:
+
+`pod install`
+
+- Download a new pod for the current project and add it to the Podfile:
+
+`pod install {{pod_name}}`
```

Comment:
Good catch. I must have misread the docs. So it seems to install new pods one must add them to the Podfile and then run plain `pod install`. From [the docs](https://guides.cocoapods.org/using/pod-install-vs-update.html#pod-install):

> Every time the `pod install` command is run — and downloads and install new pods — it writes the version it has installed, for each pods, in the `Podfile.lock` file. (...)
> When you run `pod install`, it only resolves dependencies for pods that are not already listed in the Podfile.lock.

Do you agree with my reading? If so, I'll make the change accordingly.